src/components/provider.js

```diff
@@ -9,21 +9,3 @@ export function provider(store) {
     return { connect: _connector.connect, ...store};
```

Comment:
If we made the provider `@Injectable`, would we be able to do

```ts
export default class SomeComponent {
    constructor(private ngRedux: ngRedux // or INgRedux or whatever ...
}
```

instead of having to use `@Inject('ngRedux')` ?